pages/dkp/kaptain/1.3.0/upgrade/index.md

```diff
@@ -34,40 +33,71 @@ Before you begin, ensure you have:
           enabled: true
     ```
 
-* Ensure the Kaptain addon repository is present in your Konvoy `cluster.yaml`:
+- Ensure the Kaptain addon repository is present in your Konvoy `cluster.yaml`:
+  
   ```yaml
       - configRepository: https://github.com/mesosphere/kubeaddons-kaptain
         configVersion: stable-1.20-1.4.0
         addonsList:
           - name: knative
             enabled: true
   ```
+  
+## Upgrade Kaptain 
+
+1. Perform the upgrade:
+
+### For Konvoy 1.x: 
+
+Add the following properties to your `params.yaml` file: 
+
+  ```bash
+  dkpPlatformVersion: "1" 
+  installMinioOperator: "true"
+  ```
+
+And upgrade Kaptain: 
+
+  ```bash
+  kubectl kudo upgrade --instance kaptain --namespace kubeflow ./kubeflow-1.4.0_1.3.0.tgz -P params.yaml
+  ```
+  
+### For Konvoy 2.x:
+
+Upgrade Kaptain:
 
-* [Download][download] the `kubeflow-1.4.0_1.3.0.tgz` tarball from the support website.
-* Upgrade Kaptain:
   ```bash
   kubectl kudo upgrade --instance kaptain --namespace kubeflow ./kubeflow-1.4.0_1.3.0.tgz
   ```
-* Monitor the upgrade process by running:
+
+2. Monitor the upgrade process by running:
```

Comment:
```suggestion
1.  Monitor the upgrade process by running:
```

Reply:
Markdown dictates DO NOT use sequential numbers, always use "1.  " (note the two blank spaces)